tests/test_freqtradebot.py

```diff
@@ -1606,7 +1631,7 @@ def test_exit_positions(mocker, default_conf, limit_buy_order, caplog) -> None:
     assert n == 0
     # Test amount not modified by fee-logic
```

Comment:
What is this test testing? get_real_amount is not relevant for this test (as handle_trade is mocked).